plugins/millumin/README.md

```diff
@@ -0,0 +1,4 @@
+# Millumin Plugin
+
+Output fixture to be included in Millumin 2 (http://www.millumin.com)
+To install the fixtures, copy the downloaded files into : '/Users/<username>/Library/Millumin/Light-Fixtures' and restart Millumin
```

Comment:
Are subfolders allowed?

Also, please try to use the same Markdown structure as the other plugins.